src/LibVulkan.jl

```diff
@@ -27,7 +27,8 @@ function __init__()
     libvulkan_handle[] = Libdl.dlopen(libname)
 end
 
-using CEnum
+include(joinpath(@__DIR__, "..", "gen", "CEnum.jl"))
```

Comment:
maybe it's better to move `CEnum.jl` to the src folder as it's not generated by Clang.jl.